src/pages/home/getting-started/configure-datastreams.md

```diff
@@ -46,4 +46,174 @@ The following steps illustrate how to create a sample datastream for a mobile im
 
 Your datastream is ready to be used now. If different settings are required per environment, create new datastreams with the respective configuration settings.
 
-For additional information on datastreams in the Data Collection UI, see the [configuring a datastream guide](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/initial-configuration/create-datastream.html).
+## Sending overrides to the Edge Network
+
+After configuring the datastream overrides, you can now send these overrides to the Edge Network through Mobile SDK. Sending the overrides using the `sendEvent` API is the final step of activating the datastream configuration overrides.
+
+### Datastream ID override
+
+The following examples demonstrate what a datastream ID override could look like in a Mobile SDK integration:
+
+<CodeBlock slots="heading, code" repeat="2" languages="iOS (Swift), Android (Kotlin)" />
```

Comment:
are we no longer extracting code blocks in separate files?

Reply:
I think for simple configurations (like this one, where we don't need to show both the request/response), using code blocks like this is a little more elegant - see the attached video:

https://github.com/AdobeDocs/aep-mobile-sdkdocs/assets/74737634/671cf530-6409-43f3-aa78-4c45f6112ae6

Which is why I'm leaning more towards this direction for things like this. (I haven't figured out how to do the more complex ones where we do the different flows in these blocks, hence me leaning one direction for each).